HousingRegisterApi/V1/Gateways/NotifyGateway.cs

```diff
@@ -19,7 +19,7 @@ public NotificationResponse SendVerifyCode(Applicant resident, string verifyCode
         {
             var personalisation = new Dictionary<string, object>
             {
-                {"resident_name", resident.Person.FirstName},
+                {"resident_name", ""},
```

Comment:
Personalisation token should be removed from the gov notify template if it's not being used.